lib/std/uuid.zig

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2020 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+
+const rand = @import("rand.zig");
+const mem = @import("mem.zig");
+
+pub const UUID = struct {
+    id: [36]u8,
+};
+
+//Creates a brand new UUID v4 object given a random number generator and allocator.
+//Format: xxxxxxxx-xxxx-4xxx-xxxx-xxxxxxxxxxxx
+//Allocated object must be destroyed by the user.
+pub fn newv4(alloc: *mem.Allocator, gen: *rand.Random) !*UUID {
```

Comment:
This should be a method on the UUID object. However, I think this whole file should become the UUID struct.